tools/editor/plugins/script_editor_plugin.cpp

```diff
@@ -288,6 +288,7 @@ void ScriptTextEditor::_load_theme_settings() {
 
 	get_text_edit()->set_custom_bg_color(EDITOR_DEF("text_editor/background_color",Color(0,0,0,0)));
 	get_text_edit()->add_color_override("font_color",EDITOR_DEF("text_editor/text_color",Color(0,0,0)));
+	get_text_edit()->add_color_override("line_number_color",EDITOR_DEF("text_editor/line_number_color",Color(0,0,0)));
```

Comment:
@akien-mga `Color(0,0,0)` is the default value, if none is found in the settings, while `Color::html(…)` is saved in the settings.